docs/section22/Rule22-11.md

```diff
@@ -0,0 +1,46 @@
+
+# CHW&CW - Rule 22-11  
+
+**Rule ID:** 22-11  
+**Rule Description:** For Baseline chilled-water system that does not use purchased chilled water, variable-flow secondary pump shall be modeled as 13W/gpm at design conditions.  
+**Rule Assertion:** B-RMR = expected value  
+**Appendix G Section:** Section 22 CHW&CW Loop  
+**Appendix G Section Reference:** Section G3.1.3.10 Chilled-water supply temperature reset (System 7, 8, 11, 12 and 13)  
+
+**Applicability:** All required data elements exist for B_RMR  
+**Applicability Checks:**  
+
+1. B-RMR is modeled with at least one air-side system that is Type-7, 8, 11, 12, or 13.
+2. B-RMR is modeled with at least one chilled water system that does not use purchased chilled water.
+
+**Manual Check:** None  
+**Evaluation Context:** Building  
+**Data Lookup:** None  
+**Function Call:** None  
+
+**Applicability Checks:**  
+
+1. B-RMR is modeled with at least one air-side system that is Type-7, 8, 11, 12, or 13: `PLACEHOLDER`
+2. B-RMR is modeled with at least one chilled water system that does not use purchased chilled water: `if B_RMR.ASHRAE229.chillers:`
+
+## Rule Logic:  
+
+- For each chiller in B_RMR: `for chiller_b in B_RMR.ASHRAE229.chillers:`
+
+  - Get cooling loop that chiller servers: `chw_loop_b = chiller_b.cooling_loop`
+
+    - If cooling loop has not been saved in CHW loop array: `if chw_loop_b NOT in chw_loop_check_array:`
+
```

Comment:
Perhaps it would be easier to have a function that returns primary CHW loop and secondary CHW loop. There should be one primary and one secondary loop; the primary loop is referenced by chillers; the secondary loop is the child loop of that primary loop.